app/assets/stylesheets/urls.scss

```diff
@@ -1,3 +1,19 @@
 // Place all the styles related to the Urls controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+.hidden {
+  display: none;
+}
+
+.show {
+  display: block;
+}
+
+.colorize {
+  color: #ff0000;
+}
+
+.uncolorize {
+  color: #800080;
```

Comment:
Color literals like `#800080` should only be used in variable declarations; they should be referred to via variable everywhere else.